site/content/en/docs/Guides/Client SDKs/nodejs.md

```diff
@@ -28,14 +28,24 @@ Add the agones dependency to `package.json`, replacing with the download locatio
 }
 ```
 
-To begin working with the SDK, create an instance of it. This will open a connection to the SDK server.
+To begin working with the SDK, create an instance of it. {{< feature expiryVersion="0.12.0" >}}This will open a connection to the SDK server.{{< /feature >}}
 
 ```javascript
 const AgonesSDK = require('agones');
 
 let agonesSDK = new AgonesSDK();
 ```
 
+{{% feature publishVersion="0.12.0" %}}
+To connect to the SDK server, either local or when running on Agones, run the `sdk.connect()` method.
+This will block for up to 30 seconds if the SDK server has not yet started. If the connection cannot be made,
```

Comment:
Minor but would call out that it's an `async` method that will `resolve` once connected or `reject` on error or if no connection can be made after 30 seconds. Technically it will block execution of the calling code if that uses `await` but does not block the Node.js event loop. I try to stray away from using the term block for async code and would reserve that for synchronous code that does block the event loop e.g. heavy calculation/loop

Reply:
Thanks! Would definitely like to use more canonical language for node.js - will make this change.

Reply:
Done!